docs/live/apply.md

```diff
@@ -40,4 +30,89 @@ Flags:
+|---------------------|----------------------------|---------------------------|
+|Usage                | kubectl apply -f /dir      | kpt live apply /dir       |
+|Applied resource set | Not tracked                | Tracked                   |
+|Prune                | Imperative and error prone | Declarative and reliable  |
+|wait-for-reconcile   | Not supported              | Supported                 |
+
+Usage:
+  In terms of usage, both `kubectl apply` and `kpt live apply` follow similar pattern.
+  The user experience remains unchanged.
+
+Applied resource set(ARS):
+  `kpt live apply` tracks the state of your applied resource set and related configuration. This
+  helps `kpt` to reliably reconcile the real world ARS with your configuration changes.
+
+Prune:
+  `kpt live apply` can declaratively delete the resources which are not part of your
+  ARS anymore. `kubectl apply` also has a similar functionality with --prune
+  flag. However, it heavily depends on labels provided by user, which is imperative and
+  error prone. On the other hand, prune is default option for `kpt live apply` and app
+  state is completely managed and tracked by kpt. The only additional step users should
+  perform in order to enjoy the benefits of prune is to add the grouping object ConfigMap
+  to your package with UNIQUE label.
+
+wait-for-reconcile:
+  `kubectl apply` will simply apply the resources. Users must write their own logic
+  to poll using `kubectl get` to check if the applied resources were fully reconciled.
+  `kpt live apply` can wait till the resources are fully reconciled with continuous status
+  updates.
+
+#### Prune Example 
+Consider an example of a directory with three resources which should be applied to
+the cluster. This is a simple use case and both `kpt live apply` and `kubectl apply`
+work in the same way.
+
+```
+service-1 (Service)
+deployment-1 (Deployment)
+config-map-1 (ConfigMap)
+```
+
+So far so good, but imagine the package is updated to contain the following resources,
+including a new ConfigMap named `config-map-2` (Notice that `config-map-1`
+is not part of the updated package). This means the user intent is to delete `config-map-1`
+and create `config-map-2`
+
+```
+service-1 (Service)
+deployment-1 (Deployment)
+config-map-2 (ConfigMap)
+```
+
+At this point, it gets tricky with `kubectl apply` to achieve the task. As kubectl doesn't
+store the information that `config-map-1` belongs to this applied resource set, it just creates
+`config-map-2` and doesn't delete `config-map-1`. Alternative options with kubectl to prune
+resources is to use --prune flag specifying label or by prune-whitelist which are very dangerous
+as it might delete other resources not related to this resource set sharing same cluster. On the 
+other hand, `kpt live apply` tracks the state of Applied resource set and when the updated package
+is applied, `config-map-1` is automatically deleted (pruned).
+
+##### Inventory ConfigMap
```

Comment:
We are referring to both inventory configmap and grouping object configmap in this text. It is confusing. I think the previous description that described the grouping object template and the grouping objects was clearer.

Reply:
I agree